### DISCUSSION TITLE
Using budibase for new development

### DISCUSSION THREAD

**Discussion by tgtwm1github** [Budibase/budibase#1744] [Help] [answered]
Hello,

I am a product architect interested in using budibase for a new product I am building.

The product is being designed to fully leverage aws services. The original intent was to use react for you but I would like to try budibase.

Please let me know if I can ask a few questions to make sure I am on the right track.

I would like to use aws cognito for user management, aws lambda serverless for app logic and postgres for the db.

Will I be able to use budibase in this stack mix as my UI tool?
Also, the UI will be customized based on user localization for date display, currency display , themes, etc. Are these possible?

**Comment by shogunpurple** [comment, marked as answer]
@tgtwm1github Budibase is generally built for creating internal tools in an organisation or business, rather than products.

You would not be able to use cognito for user management right now. Budibase provides its own auth providers (currently local auth and google at this time).

You can interact directly with postgres and REST APIs (if your lambda is behind API gateway for example).

Customising the user localization for date display and more is not something that is currently supported either.

**Reply by tgtwm1github** [reply]
Appreciate the crisp, clear answer. Saved me a lot of time and helps me a lot in planning.

I'll still try the product and keep tabs...

Good luck
